.travis.yml

```diff
@@ -1,14 +1,22 @@
+---
 language: python
 
 python:
-    - "2.7"
+  - 2.7
 
-install: "make"
+install:
+  - pip install -r requirements.txt
+  - pip install yamllint pep8
```

Comment:
probably not - there's possibly a smarter way to specify dev dependencies in requirements.txt